### PR TITLE
Update repo.rb with version '1.21'

### PR DESCRIPTION
**Pull request by ChaseVoid**
updated URL and SHA1 to correct version.

### CODE REVIEW

File: Library/Formula/repo.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage 'http://source.android.com/source/git-repo.html'
-  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.13/repo'
+  url 'https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo'
   version '1.21'
```
Comment (DomT4) [inline Library/Formula/repo.rb]: Presumably this needs bumping as well?

Comment (ChaseVoid) [inline Library/Formula/repo.rb]: The stable version currently is 1.21 but the url was pointing to incorrect
version of 1.13

Comment (DomT4) [inline Library/Formula/repo.rb]: I see. Maybe it's worth adding `revision 1` to the formula so people who think they're currently using version `1.21` actually are?

Comment (ChaseVoid) [inline Library/Formula/repo.rb]: Will this help?

Comment (DomT4) [inline Library/Formula/repo.rb]: `1` doesn't need to go inside `""`. Should just be:

```ruby
sha1 "9421408dba2d80e3c73910567f70f2e0c91dab2f"
revision 1
```

Thanks!

**Comment by ChaseVoid** [comment]
Thanks. I've updated it.

File: Library/Formula/repo.rb (hunk 1 above)
Comment (ChaseVoid) [inline Library/Formula/repo.rb]: Thanks, I've updated the pull request.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/c5dcbb8d621e12bc12f67f41eea5c6b3c0a0d42c. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Small nit for future: Commit messages for non-version bumps should be styled `repo: use correct url for version` or similar, Thanks!

**Comment by ChaseVoid** [comment]
Will keep that in mind. Thanks :)

**Pull request closed without merging**